mls-rs-uniffi/src/config.rs

```diff
@@ -0,0 +1,43 @@
+use std::{fmt::Debug, sync::Arc};
+
+use mls_rs::{
+    client_builder::{self, WithGroupStateStorage},
+    identity::basic,
+};
+use mls_rs_core::error::IntoAnyError;
+use mls_rs_crypto_openssl::OpensslCryptoProvider;
+
+use self::group_state::GroupStateStorageWrapper;
+
+mod group_state;
+
+#[derive(Debug, thiserror::Error, uniffi::Error)]
+#[uniffi(flat_error)]
+#[non_exhaustive]
+pub enum FFICallbackError {
```

Comment:
Did you consider just extending the `Error` enum instead to keep the API smaller?

Reply:
We can do that as a follow up, was keeping this isolated and didn't merge them together.